3.3/.sti/bin/assemble

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+source scl_source enable python33
+set -e
+
+echo "---> Installing application source"
+cp -Rf /tmp/src/* ./
+
+echo "---> Building your Python application from source"
+if [ -f setup.py ]; then
+    python setup.py install --user
+elif [ -f requirements.txt ]; then
```

Comment:
@bparees not sure what you mean by that. `setup.py` is a complete information about your package/application (including name, author and deps). In that case using `requirements.txt` doesn't make sense as in most cases its contents will be already read into `setup.py`. In all other cases I'm just falling back to `requirements.txt` if it exists for a list of deps.